public/js/components/App.css

```diff
@@ -6,7 +6,7 @@
 
 :root.theme-light,
 :root .theme-light {
-  --theme-search-overlays-semitransparent: rgba(221, 225, 228, 0.66);
```

Comment:
hmm, i don't think we'll want to change this. It's a global that's shared in many places.

I think we can likely just use ` rgba(190, 190, 190, 0.8);` for the box-shadow. the menu is special in that it's only for the local environment and...